doc_template/index.rst

```diff
@@ -119,6 +119,11 @@ The Allen SDK provides Python code for accessing experimental metadata along wit
 See the `mouse connectivity section <connectivity.html>`_ for more details.
 
 
+What's New - 2.10.2
+-----------------------------------------------------------------------
+- Adds restriction to disallow hdmf version which breaks pynwb
```

Comment:
I would reword this to:
`- Adds restriction to disallow hdmf version 2.5.0 which accidentally introduced a major version breaking change`

Reply:
Ok done